cli/cmd/plugin/standalone-cluster/packages/packages.go

```diff
@@ -312,7 +319,12 @@ func (am *PackageClient) CreateRootServiceAccount(ns, name string) (*v1.ServiceA
 }
 
 func (am *PackageClient) ListPackagesInNamespace(ns string) ([]datapackaging.Package, error) {
-	pkgList, err := am.pcs.DataV1alpha1().Packages(ns).List(context.TODO(), metav1.ListOptions{})
+	pkgList := &datapackaging.PackageList{}
+	err := am.arc.Get().
```

Comment:
Pulled this PR down and I'm getting the following compile error:
```
github.com/vmware-tanzu/community-edition/cli/cmd/plugin/standalone-cluster/packages
packages/packages.go:323:11: am.arc undefined (type *PackageClient has no field or method arc)
```

did this actually need to be `aggRestClient`?:
```diff
- err := am.arc.Get().
+ err := am.aggRestClient.Get().
```

Reply:
oops! very weird 🤔